create-instances.py

```diff
@@ -14,10 +15,13 @@
 
 args = parser.parse_args()
 
-ec2 = boto3.client('ec2')
+ec2 = boto3.client('ec2', region_name=args.region)
 #response = ec2.describe_instances()
 print 'Going to request an on-demand EC2 instance...'
 
+if args.disk_size >= 2048 or args.disk_size < 128:
+    raise Exception('Disk size should be in [128, 2048)')
```

Comment:
need more reason. You can refer to http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSVolumeTypes.html 
"Linux AMIs require GPT partition tables and GRUB 2 for boot volumes 2 TiB (2048 GiB) or larger. Many Linux AMIs today use the MBR partitioning scheme, which only supports up to 2047 GiB boot volumes. If your instance does not boot with a boot volume that is 2 TiB or larger, the AMI you are using may be limited to a 2047 GiB boot volume size. Non-boot volumes do not have this limitation on Linux instances."